README.md

```diff
@@ -11,6 +11,21 @@ variables. You will need to set F90 (and optionally, F90FLAGS).
 At link time you will need to link with the OpenCL run-time
 libraries on your system.
 
+# Controlling runtime behaviour"
+
+There are 3 environment variables to control the OpenCL execution
+at runtime:
+
+- `FORTCL_KERNELS_FILE`: Specify at run-time the binary or source file
+where the OpenCL kernels are if this is not given at compile-time.
+
+- `FORTCL_PLATFORM`: If a system has multiple OpenCL platforms, this
```

Comment:
Need to say something like "is an integer that corresponds to one of the OpenCL platforms listed by running....".

Reply:
Done